product-scenarios/scenarios-common/src/main/java/org/wso2/am/scenario/test/common/APIStoreRestClient.java

```diff
@@ -1052,7 +1052,33 @@ public HttpResponse getAPIListFromStoreAsAnonymousUser(String tenantDomain)
         }
     }
 
+    /**
+     * Get tags of APIs in Store for an anonymous user
+     *
+     * @return HttpResponse - Response with tags of APIs visible to anonymous users
+     * @throws APIManagerIntegrationTestException
+     */
+    public HttpResponse getTagListFromStoreAsAnonymousUser()
+            throws APIManagerIntegrationTestException {
+        try {
+            HttpResponse httpResponse = HttpClient.doPost(
+                    new URL(backendURL + "/site/blocks/tag/tag-cloud/ajax/list.jag?action=getAllTags" +
+                            "&tenant=carbon.super"),
+                    "", requestHeaders);
+
+            if (new JSONObject(httpResponse.getData()).getBoolean("error")) {
+                throw new APIManagerIntegrationTestException("Error when getting tag list as anonymous user");
```

Comment:
let's not throw APIManagerIntegrationTestException instead just throw Exception

Reply:
Fixed